packages/storybook/stories/CallWithChatComposite/CallWithChatCompositeDocs.tsx

```diff
@@ -45,9 +47,13 @@ export const Docs: () => JSX.Element = () => {
 
     if (url.includes('adding-file-sharing') && refFileSharing.current) {
       scrollToRef(refFileSharing);
-    } else if (url.includes('in-teams-interop-meeting-chat-thread') && refFileSharingTeams.current) {
+    } else if (url.includes('adding-image-sharing') && refImageSharing.current) {
+      scrollToRef(refImageSharing);
+    } else if (url.includes('inline-image-in-teams-interop-meeting-chat-thread') && refImageSharingTeams.current) {
+      scrollToRef(refImageSharingTeams);
+    } else if (url.includes('file-sharing-in-teams-interop-meeting-chat-thread') && refFileSharingTeams.current) {
       scrollToRef(refFileSharingTeams);
-    } else if (url.includes('in-azure-communication-services-chat-thread') && refFileSharingACS.current) {
+    } else if (url.includes('file-sharing-in-azure-communication-services-chat-thread') && refFileSharingACS.current) {
```

Comment:
Does this mean the original URL is no longer valid?

Reply:
yes, but MSDoc has been updated few weeks ago and no longer pointing to this tag